### ISSUE TITLE
 XmlDoc is displayed along with the summary tag

### ISSUE THREAD

**Issue by DedSec256** [dotnet/fsharp#11611]
Rider uses `FSharpXmlDoc.FromXmlText(xmlDoc) -> xmlDoc.UnprocessedLines` to get xml documentation text.
And if earlier for library entities there was a summary inner text there, then, apparently after https://github.com/dotnet/fsharp/pull/11454, `UnprocessedLines` contains text along with the summary tag, which looks like a regression.

Rider 2021.1 (old behavior)
![image](https://user-images.githubusercontent.com/26364714/120023146-13cd1700-bff6-11eb-9fb0-171e333aa1ee.png)

Rider 2021.2 Dev (new behavior)
![image](https://user-images.githubusercontent.com/26364714/120023194-20ea0600-bff6-11eb-8173-dd4afe75afa2.png)

It seems to me it has something to do with how the documentation is read here
https://github.com/dotnet/fsharp/pull/11454/files#diff-56fbb9330ceb81afbfa2ea19c8b4bb3946a732f8ffb128f0d1435884808e06e8R250

**Comment by auduchinok** [comment]
Just to note, Rider hadn't used the `UnprocessedLInes` member before #11454. I'd expect `UnprocessedLInes` to indeed contain the tags (since the lines are not expected to be processed in any meaningful name, and it seems to behave correctly), but I'd expect there to be another member that preserves the previous behaviour, i.e. cleans the doc.

**Comment by cartermp** [comment]
Yeah, we seemed to regress something here. We don't see this in VS though. I know our tooltips code is pretty complex though. Would you be able to use the approach we do?

**Comment by auduchinok** [comment]
> Would you be able to use the approach we do?

Yup, we're going to do a similar processing on our side too, it's just seemed reasonable to report the regression in case there're other FCS clients might be using the APIs in question.
